tailwind.config.js

```diff
@@ -0,0 +1,121 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {
+      backgroundImage: {
+        "hero-underline-1": "url(/assets/icons/small-line.svg)",
+        "hero-underline-2": "url(/assets/icons/large-line.svg)",
+      },
+      backgroundPosition: {
+        under: "bottom 0px right 15px",
+      },
+      colors: {
+        gray: {
+          1: "#E5E5E5",
+          2: "#CBCBCB",
+          3: "#B1B1B1",
+          4: "#979797",
+          5: "#7D7D7D",
+          6: "#636363",
+          7: "#494949",
+          8: "#2E2E2E",
+          9: "#141414",
+        },
+        secondary: "#9A9A9A",
+      },
+      fontSize: {
+        "custom-xs": "0.938rem",
```

Comment:
i think its not necessary to prefix custom. may be you can overlide the default one.